lib/web/apiserver.go

```diff
@@ -2086,21 +2114,15 @@ func (h *Handler) siteSessionsGet(w http.ResponseWriter, r *http.Request, p http
 		return nil, trace.Wrap(err)
 	}
 
-	sessions, err := clt.GetSessions(apidefaults.Namespace)
+	trackers, err := clt.GetActiveSessionTrackers(r.Context())
```

Comment:
nit: it would be nice to have a type parameter  in `GetActiveSessionTrackers` and filter out unnecessary sessions before sending them over a network.

Reply:
Agreed, a option system with some filters could be useful, but feels out of scope for this PR.